src/inbox/mod_inbox.erl

```diff
@@ -91,8 +100,8 @@ start(Host, Opts) ->
 stop(Host) ->
     mod_disco:unregister_feature(Host, ?NS_ESL_INBOX),
     mod_inbox_muc:stop(Host),
-    ejabberd_hooks:delete(user_send_packet, Host, ?MODULE, user_send_packet, 90),
-    ejabberd_hooks:delete(filter_local_packet, Host, ?MODULE, filter_packet, 90),
+    ejabberd_hooks:delete(user_send_packet, Host, ?MODULE, user_send_packet, 70),
+    ejabberd_hooks:delete(filter_local_packet, Host, ?MODULE, filter_packet, 70),
```

Comment:
This may not work. The hook was added with priority `90` and here we are deleting with priority `70`. I suggest to define all the hooks in one place and use function `ejabberd_hooks:add/1` and `ejabberd_hooks:delete/1` which take a list of hooks as input. It's easier to maintain consistency when all the handlers are defined in one place. There are modules using these function already so you can take a look how it's used.